app/src/main/java/org/mozilla/fenix/library/bookmarks/viewholders/BookmarkNodeViewHolder.kt

```diff
@@ -5,38 +5,41 @@
 package org.mozilla.fenix.library.bookmarks.viewholders
 
 import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.extensions.LayoutContainer
 import mozilla.components.concept.storage.BookmarkNode
+import mozilla.components.concept.storage.BookmarkNodeType
 import org.mozilla.fenix.library.LibrarySiteItemView
 import org.mozilla.fenix.library.SelectionHolder
 import org.mozilla.fenix.library.bookmarks.BookmarkFragmentState
 import org.mozilla.fenix.library.bookmarks.BookmarkItemMenu
 import org.mozilla.fenix.library.bookmarks.BookmarkViewInteractor
+import org.mozilla.fenix.utils.Do
 
 /**
  * Base class for bookmark node view holders.
  */
 abstract class BookmarkNodeViewHolder(
-    override val containerView: LibrarySiteItemView,
+    protected val containerView: LibrarySiteItemView,
     private val interactor: BookmarkViewInteractor
-) : RecyclerView.ViewHolder(containerView), LayoutContainer {
```

Comment:
I think we do need to implement LayoutContainer here so the views are cached properly. We also seem to have our own ViewHolder class that does that for us.

Reply:
In this case all the views are on `LibrarySiteItemView`, so `LayoutContainer` is redundant.